src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    //final String Splitter_Symbol = ",";
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
split your finctionality inside this method into 3 separate parts:
```
parseDataFromFile(fileName)
createReport(data)
writeDataToFile(report, fileName)
```

Reply:
the first method should only read data from file, createReport should parse the data